app/src/controllers/project/sagas.js

```diff
@@ -281,8 +281,7 @@ function* watchUpdatePAState() {
 
 function* fetchProject({ payload: { projectId, fetchInfoOnly } }) {
   try {
-    const id = yield select(projectIdSelector);
-    const project = yield call(fetch, URLS.project(id || projectId));
+    const project = yield call(fetch, URLS.project(projectId));
```

Comment:
Please check 52080 ticket it related issue which come back after reverting https://github.com/reportportal/service-ui/pull/2870